Configuration/TCA/tx_sfbanners_domain_model_banner.php

```diff
@@ -43,21 +38,6 @@
                 'type' => 'language',
             ],
         ],
-        'l10n_parent' => [
-            'displayCond' => 'FIELD:sys_language_uid:>:0',
-            'exclude' => 0,
```

Comment:
Only `'exclude' => 0,` must be removed. The `'l10n_parent'` must stay, else localization will not work any more.